src/components/Modals/RenameFile.test.js

```diff
@@ -11,7 +11,6 @@ describe("Testing the new file modal", () => {
     let inputBox;
     let saveButton;
     let getByText;
-    let getByRole;
```

Comment:
ah - this getByRole needs a refactor, this will fail as-is I think, see linting here